lib/lib/linkHandler.tsx

```diff
@@ -65,7 +67,7 @@ export const Link = forwardRef<HTMLAnchorElement, LinkProps>(
     const { isActive } = useNavigation()
 
     const overridenProps = {
-      "data-is-active": isActive(props.href) || undefined,
+      "data-is-active": isActive(props.href, { exact: props.exactMatch }),
```

Comment:
We should set `exactMatch` default to `false` for the sake of reducing the the optional prop pollution. If we don't stop it here, it will spread everywhere

Reply:
Not sure I follow! `exactMatch` defaults to `false` in the implementation of `isActive`:

https://github.com/factorialco/factorial-one/pull/663/files#diff-1386e11f651c9b9dd4ccd2245ad6444068c5d8176915645c3554db1d2b6ffb46R51